webview/src/shared/components/messagesMenu/styles.module.scss

```diff
@@ -17,6 +17,16 @@
   }
 }
 
+.disabledItem {
```

Comment:
[C] Based on the previous comments you will need this but right now it is not required and should not have been included in the PR.